### DISCUSSION TITLE
沒辦法安裝虛擬環境該怎麼辦

### DISCUSSION THREAD

**Discussion by dingjunwei225** [1101-GS4719-C/Python-Community-GS4719-C#24] [Q&A]
![242794612_858596358193375_5339902156894655842_n](https://user-images.githubusercontent.com/91170919/135042588-1ceafb19-3b68-4e77-99e0-7e2ca3f28677.jpg)

**Comment by chia-kai-chang** [comment]
![image](https://user-images.githubusercontent.com/1768891/135044681-c66eae83-66f7-405b-89cb-68bedf943bf9.png)

安裝套件的指令，最後面不用加點點 `.`
